package.json

```diff
@@ -1,10 +1,17 @@
 {
   "name": "focusable-selectors",
+  "type": "module",
```

Comment:
🔍 I think the absence of or presence of `type: "module"` might also affect how NPM resolves imports. Regardless, this _is_ an ES module now; we may as well treat it like one.